### PR TITLE
Re-establish pallet_revive weights baseline

### PR DESCRIPTION
**Pull request by mordamax**
- update baseline for pallet_revive
- update cmd pipeline name
- Fix compilation after renaming some of benchmarks in pallet_revive. [Runtime Dev]. Changed the "instr" benchmark so that it should no longer return to little weight. It is still bogus but at least benchmarking should not work. (by @athei )

### CODE REVIEW

**Comment by athei** [comment]
It doesn't compile because we renamed some benchmarks but never fixed the places where the benchmarks were used. The benchmarking worked fine I guess just need some follow up fixes here.

I can push the fixes to this PR later.

**Comment by athei** [comment]
So I fixed the compilation. But because the `instr` benchmark is so whacky it foes down to zero and we and up with divide by zero panics. Have to look into this later.

**Comment by athei** [comment]
I changed the `instr` benchmark so that it should no longer return to little weight. It is still bogus but at leats benchmarking should not work.

**Comment by athei** [comment]
Do you know why it didn't work?

File: .github/workflows/cmd-tests.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
+  test-cmd:
```
Comment (alvicsam) [inline .github/workflows/cmd-tests.yml:14]:
```suggestion
  test-cmd-bot:
```
I would use this name because `test-cmd` is still confusing imo

**Review by alvicsam — CHANGES_REQUESTED** [review]

**Comment by mordamax** [comment]
> Do you know why it didn't work?

```
2024-09-28T12:33:05.093879Z  WARN frame_omni_bencher: The FRAME omni-bencher is not yet battle tested - double check the results.    
2024-09-28T12:33:05.094321Z  INFO frame_benchmarking_cli::pallet::command: Loading WASM from target/release/wbuild/kitchensink-runtime/kitchensink_runtime.wasm    
2024-09-28T12:33:10.291775Z ERROR frame_benchmarking_cli::pallet::command: Please pick one of the available presets with `--genesis-builder-preset=<PRESET>`. Available presets (0): []. Error: "The preset with name Some(\"development\") is not available."    
Error: Input("The preset with name Some(\"development\") is not available.")
```

**Comment by mordamax** [comment]
bot bench substrate-pallet --pallet=pallet_revive

**Comment by mordamax** [comment]
bot bench substrate-pallet --pallet=pallet_revive

**Review by athei — APPROVED** [review]

**Review by alvicsam — APPROVED** [review]

**Review by pgherveou — APPROVED** [review]

**Comment by athei** [comment]
Why are those flaky tests required again? It is really starting to get annoying.

**Pull request merged**